src/Sylius/Bundle/ShopBundle/Resources/config/app/config.yml

```diff
@@ -312,3 +216,900 @@ sylius_ui:
+        sylius.shop.checkout.complete.after_content_header:
+            blocks:
+                legacy:
+                    template: "@SyliusUi/Block/_legacySonataEvent.html.twig"
+                    context:
+                        event: sylius.shop.checkout.complete.after_content_header
+                        
+        sylius.shop.login.after_content_header:
+            blocks:
+                legacy:
+                    template: "@SyliusUi/Block/_legacySonataEvent.html.twig"
+                    context:
+                        event: sylius.shop.login.after_content_header
+                        
+        sylius.shop.register.after_content_header:
+            blocks:
+                legacy:
+                    template: "@SyliusUi/Block/_legacySonataEvent.html.twig"
+                    context:
+                        event:   sylius.shop.register.after_content_header
+
+        sylius.shop.account.dashboard.after_content_header:
+            blocks:
+                legacy:
+                    template: "@SyliusUi/Block/_legacySonataEvent.html.twig"
+                    context:
+                        event: sylius.shop.account.dashboard.after_content_header
+                        
+        sylius.shop.request_password_reset_token.after_content_header:
+            blocks:
+                legacy:
+                    template: "@SyliusUi/Block/_legacySonataEvent.html.twig"
+                    context:
+                        event: sylius.shop.request_password_reset_token.after_content_header
+                        
+        sylius.shop.account.address_book.create.after_content_header:
+            blocks:
+                legacy:
+                    template: "@SyliusUi/Block/_legacySonataEvent.html.twig"
+                    context:
+                        event: sylius.shop.account.address_book.create.after_content_header
+                        
+        sylius.shop.account.address_book.update.after_content_header:
+            blocks:
+                legacy:
+                    template: "@SyliusUi/Block/_legacySonataEvent.html.twig"
+                    context:
+                        event: sylius.shop.account.address_book.create.after_content_header # Intentionally using "create" event for backwards compatibility
+                        
+        sylius.shop.account.address_book.index.after_content_header:
+            blocks:
+                legacy:
+                    template: "@SyliusUi/Block/_legacySonataEvent.html.twig"
+                    context:
+                        event: sylius.shop.account.address_book.index.after_content_header
+                        
+        sylius.shop.account.order.index.after_content_header:
+            blocks:
+                legacy:
+                    template: "@SyliusUi/Block/_legacySonataEvent.html.twig"
+                    context:
+                        event: sylius.shop.account.order.index.after_content_header        
```

Comment:
This new event is not using, legacy is using : https://github.com/Sylius/Sylius/pull/11143/files#diff-cd30ca237d9d140b0ba8594e7645c29eR865

and event above the same sytuation : `sylius.shop.account.order.show.after_content_header`